### PR TITLE
docs: ADR-202 for linked wearables v2

### PR DESCRIPTION
**Pull request by pentreathm**

### CODE REVIEW

**Review by marianogoldman — APPROVED** [review]

File: content/ADR-202-linked-wearables.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+## Context, Reach & Prioritization
+
+To foster greater adoption and streamline the cost and maintenance of LinkedWearables, it is necessary to implement a new process and architecture capable of adapting to the demand while preventing any potential performance issues for the client. Additionally, it will enhance the user experience by allowing users to view all their LinkedWearables directly on the backpack, without needing to pre-select collections from a dropdown menu.
+
+## Decision
+
+LinkedWearables in Decentraland are wearables exclusively granted to owners of specific NFTs. While the wearables themselves cannot be traded, the underlying NFTs can. Verifying ownership of these wearables necessitated a third-party API. To streamline this process, a new system of URNs for LinkedWearables and wearable metadata will be introduced. These URNs and metadata will comprehensively describe both the Decentraland wearable and its corresponding NFT, enabling on-chain verification of NFT ownership, thereby eliminating the reliance on third party APIs.
```
Comment (nachomazzara) [inline content/ADR-202-linked-wearables.md:26]: ```Verifying ownership of these wearables necessitated a third-party API.``` -> ```Verifying ownership of these wearables necessitated a service to provide the owner of the represented NFTs.```

File: content/ADR-202-linked-wearables.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+- Owns any item from the specified NFT collections
+- Owns any item from a list of items within a specified NFT collection
+- Owns any item from a range of items within a specified NFT collection
+
+The mappings configuration supports lists of contracts by chain and multiple sets of collection items, as long as the sets do not overlap with each other in the same collection. This flexibility can accommodate any use case required by a third party. A Decentraland wearable can be linked/mapped to single or multiple items across one or many NFT collections on different chains.
+
+The examples below are mapping configurations within a wearable entity that represent various use cases. This information is sent to the Catalyst during the deployment of the wearable entity. 
+
+- **Owns a specific NFT collection item**: the user owning the item with id `1` in the NFT collection `0x1234567890abcdef1234567890abcdef12345678` on `matic` will be granted with this wearable
+```code:json
```
Comment (nachomazzara) [inline content/ADR-202-linked-wearables.md:58]: remove `code:json` cause this block is hidden [here](https://adr.decentraland.org/adr/ADR-202)

File: content/ADR-202-linked-wearables.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+   - **contract**: Address of an ERC-721 or ERC-1155 collection contract 
+   - **tokenId**: collection item id 
+
+Example URN: `urn:decentraland:matic:collections-thirdparty:adidas:summer2024:red-hoodie:matic:0xba0c9cf4da821dba98407cc4f9c11f6c7a5f9bbc:1`
+
+With the information from the URN, the collection contract at address `0xba0c9cf4da821dba98407cc4f9c11f6c7a5f9bbc` on `matic`, can be queried to verify if the user equipping the wearable owns the item with id `1`.
+
+### Third Party Registry 
+
+When a third party is registered, a record is added to the third party's smart contract. This record must now include the list of NFT collection contracts and their corresponding chains for which the third party will create wearables. This information is essential for the Catalyst nodes to create a backpack endpoint, retrieve all NFTs owned by a wallet across those collections, and identify any corresponding linked wearables for those NFT items.
```
Comment (nachomazzara) [inline content/ADR-202-linked-wearables.md:164]: Can we add an example?

Comment (nachomazzara) [reply content/ADR-202-linked-wearables.md:164]: It should be

`tp:version:lw-name:lw-description:lw-collections`

E.g: `tp:1:third-party-test:this-is-a-description:sepolia-0x74c78f5A4ab22F01d5fd08455cf0Ff5C3367535C;amoy-0x31c78f3A4ac21F41e5ad28345df1Fa5A3367213A`

File: content/ADR-202-linked-wearables.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+## Deadline
+
+Once the new implementation of LinkedWearables is fully supported, there will be a migration period to re-upload all existing LinkedWearables with the necessary mapping information. A deadline will be established for this migration. After the migration period ends and the deadline is reached, the old Third Party Resolvers API will no longer be used by the Catalyst nodes.
+
+Date: TBD
+
+## Consequences
+
+- Third parties will no longer need to pay for the development and maintenance of an API to verify NFT ownership and the associated wearables.
```
Comment (nachomazzara) [inline content/ADR-202-linked-wearables.md:208]:
```
Third parties will no longer need to pay for the development and maintenance of an API to verify NFT ownership and the associated wearables.
```

to


```
Third parties will no longer need to develop and maintain an API to verify NFT ownership and the associated Decentraland wearables.
```

File: content/ADR-202-linked-wearables.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+- Retrieving a profile or batch of profiles from the Catalyst node will no longer be affected by slow third-party resolvers.
+- Old third-party wearables need to be updated by the third parties to include the mapping information, or after the deadline, they will stop working, and their NFT holders won't be able to retrieve their wearables.
+- There is a dependency on the Alchemy API or similar third-party services to query which NFT items a user owns from a collection. - Only the chains supported by these services can be used. This dependency could be replaced or expanded to add support for more chains as they become available in the market.
```
Comment (nachomazzara) [inline content/ADR-202-linked-wearables.md:211]: ```There is a dependency on the Alchemy API or similar third-party services to query which NFT items a user owns from a collection. - Only the chains supported by these services can be used. This dependency could be replaced or expanded to add support for more chains as they become available in the market.```

to

```
For this iteration, the service to validate NFT ownership will be a thirdparty API. Only the chains supported by these services can be used. This service can be replaced or expanded to support more chains as they become available in the market.
```

**Pull request merged**